src/pages/patientView/PatientViewPage.tsx

```diff
@@ -41,6 +44,13 @@ export interface IPatientViewPageProps {
 
 export type MrnaRankData = { [sampleId:string]: { [entrezGeneId:string]: {percentile:number, zScore:number}}};
 export type MutSigData = { [entrezGeneId:string]:{ qValue:number } }
+export type VariantCountData = {
```

Comment:
our current lint rules favor interface over type literals, but I am not sure that if we want to stick to that convention.